src/B0Preshower_geo.cpp

```diff
@@ -210,9 +192,7 @@ static Ref_t create_B0Preshower(Detector& description, xml_h e, SensitiveDetecto
             PlacedVolume sens_pv = sensVols[ic];
             DetElement   comp_elt(r_module, sens_pv.volume().name(), mod_num);
             comp_elt.setPlacement(sens_pv);
-            // std::cout << " adding ACTS extension" << "\n";
-            Acts::ActsExtension* moduleExtension = new Acts::ActsExtension("XZY");
-            comp_elt.addExtension<Acts::ActsExtension>(moduleExtension);
```

Comment:
Do we need this here?
```
          auto &params = DD4hepDetectorHelper::ensureExtension<dd4hep::rec::VariantParameters>(comp_elt);
          params.set("axis_definitions", "XZY");
```

Reply:
Nothing in B0 is integrated with the main tracking. Will deal with these with a later PR.

Reply:
Also preshower I think is deprecated and replaced by the B0 ECal. Moving it to deprecated folder.

Reply:
Ok, I'm just pointing this out since it removes code but doesn't replace it with equivalent code. It may lead to difficulties in the future when someone adapts this code, and isn't aware of the past "XZY" axis definitions. But I leave it up to you to decide how to deal with this.

Reply:
Btw is there a deprecated folder anymore? Seems like everything is in the same space. Would comment at the top of the constructor like "Deprecated. ACTS tracking interface not functional." do?

Reply:
Sure, I think that's probably sufficient.